packages/web/src/components/gcds-error-summary/gcds-error-summary.tsx

```diff
@@ -205,13 +207,17 @@ export class GcdsErrorSummary {
               Object.keys(errorQueue).map(key => {
                 return (
                   <li class="summary__listitem">
-                    <a
-                      onClick={e => this.focusElement(e, key)}
-                      class="summary__link"
-                      href={key}
+                    <gcds-link
```

Comment:
Since you are replacing the `a.summary__link` with the `gcds-link` component, can you also please remove the CSS for `.summary__link` and those tokens?